src/types.rs

```diff
@@ -271,7 +272,7 @@ impl Id {
     let sign_arr = &pub_sign_key.0;
     let asym_arr = &pub_asym_key.0;
 
-    let mut arr_combined = [0u8; 64 * 2];
+    let mut arr_combined = [0u8; 64 * 2 + SIGNATUREBYTES];
```

Comment:
I'm wondering why we use the number 64 here. If I understand it correctly `arr_combined` consists of:

[PublicKey|PublicKey|Signature]

The first PublicKey I think is from [sodiumdioxide/crypto/sign/ed25519.rs file](https://github.com/dnaq/sodiumoxide/blob/master/src/crypto/sign/ed25519.rs#L11) and the second one is from [sodiumdioxide/crypto/asymmetricbox/curve25519xsalsa20poly1305.rs file](https://github.com/dnaq/sodiumoxide/blob/4429e27ae6359434e01d088f7dc6351a90e5e070/src/crypto/asymmetricbox/curve25519xsalsa20poly1305.rs#L12). Both I believe have size 32 bytes which would make the `64*2` calculation incorrect.

Ideally it would be written something like:
[0u8; sign::PUBLICKEYBYTES + asymmetricbox::PUBLICKEYBYTES + SIGNATUREBYTES]